<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1366</width>
      <height>768</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../res.qrc">:/img/res/background.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_exit">
    <property name="geometry">
     <rect>
      <x>1308</x>
      <y>14</y>
      <width>42</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_new_game">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>353</y>
      <width>239</width>
      <height>67</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_load_game">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>433</y>
      <width>239</width>
      <height>67</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_save_game">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>513</y>
      <width>239</width>
      <height>67</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_about">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>593</y>
      <width>239</width>
      <height>67</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_first_move">
    <property name="geometry">
     <rect>
      <x>1073</x>
      <y>641</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_previous_move">
    <property name="geometry">
     <rect>
      <x>1117</x>
      <y>641</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_next_move">
    <property name="geometry">
     <rect>
      <x>1161</x>
      <y>641</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_last_move">
    <property name="geometry">
     <rect>
      <x>1204</x>
      <y>641</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGraphicsView" name="board">
    <property name="geometry">
     <rect>
      <x>328</x>
      <y>64</y>
      <width>638</width>
      <height>638</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/img/res/empty.png)</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="blackCapturedScene">
    <property name="geometry">
     <rect>
      <x>328</x>
      <y>6</y>
      <width>638</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="whiteCapturedScene">
    <property name="geometry">
     <rect>
      <x>328</x>
      <y>721</y>
      <width>638</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px</string>
    </property>
   </widget>
   <widget class="QLabel" name="time_white">
    <property name="geometry">
     <rect>
      <x>1039</x>
      <y>90</y>
      <width>241</width>
      <height>77</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Digital Counter 7</family>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : #efefef</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="lineWidth">
     <number>4</number>
    </property>
    <property name="text">
     <string>00:00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="time_black">
    <property name="geometry">
     <rect>
      <x>1039</x>
      <y>190</y>
      <width>241</width>
      <height>78</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Digital Counter 7</family>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #7f7f7f
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="lineWidth">
     <number>4</number>
    </property>
    <property name="text">
     <string>00:00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGraphicsView" name="notationScene">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>331</y>
      <width>239</width>
      <height>304</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
